verified-reviews/utils/client.ts

```diff
@@ -167,11 +167,50 @@ export const createClient = (params: ConfigVerifiedReviews | undefined) => {
     }
   };
 
+  const storeReview = async () : Promise<Review[] | null> => {
+    try {
+      const response = await fetchAPI<Reviews['reviews']>(`https://cl.avis-verifies.com/br/cache/8/6/a/${idWebsite}/AWS/WEBSITE_API/reviews.json`, {
+        method: "GET",
+      });
+
+      return(
+        response 
+        ?
```

Comment:
I would avoid doing data transformations on clients. Clients should serve only as an interface for making http calls, the rationale behind this is because you can reuse clients across multiples actions and loaders without necessarily doing data transformations. 

If you can, move this code to the loader implementation instead

Reply:
Sure. If I understood correctly, I fixed that now in this new commit, can you validate it for me, please?